client/my-sites/themes/themes-selection.jsx

```diff
@@ -60,11 +86,17 @@ const ThemesSelection = React.createClass( {
 	},
 
 	onTierSelect( { value: tier } ) {
+		trackClick( 'search bar filter', tier );
```

Comment:
Once in, especially for a long time, it's better to just stick with the analytics track name we used. I just checked again but I wasn't able to find a way to merge two events on GA side.

Let's ping @apeatling to be sure. :)